opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -243,6 +243,8 @@ namespace Opm {
                 residual_norms_history_.clear();
                 current_relaxation_ = 1.0;
                 dx_old_ = 0.0;
+                convertInput( iteration, reservoir_state, ebosSimulator_ );
+                ebosSimulator_.model().invalidateIntensiveQuantitiesCache(/*timeIdx=*/0);
             }
```

Comment:
The convertInput() method is a method of this class, and I'm pretty sure I should not call it from outside, so I am sure I understand where you think I should move this?

Other than that, it does fix an inconsistency for "normal" first steps as well, so I think eventually something like this should go in.